### ISSUE TITLE
Requesting a new release to fix Rails 6 deprecation warning

### ISSUE THREAD

**Issue by nvolker** [redis-store/redis-activesupport#126]
Hey there! 😺 

I was wondering if there could possibly be a new release of this lovely little gem? Possibly `v5.3.0` or `v.5.2.1`. Our Rails `6.0.3.6` application has the following deprecation warning (from the latest release; `v5.2.0`):
```
/{...}/.rvm/gems/ruby-2.7.1@{...}/gems/redis-activesupport-5.2.0/lib/active_support/cache/redis_store.rb:85:
warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
```

I noticed this has since been updated on `master` ([master code link](https://github.com/redis-store/redis-activesupport/blob/master/lib/active_support/cache/redis_store.rb#L86) versus [current v5.2.0 release code link](https://github.com/redis-store/redis-activesupport/blob/v5.2.0/lib/active_support/cache/redis_store.rb#L85)).

**Comment by tubbo** [comment]
if you use Rails 6, there’s a built in cache store called :redis_cache_store so you don’t really need this gem, fwiw.   
  
  
  
    
  

  
  
>

**Comment by tubbo** [comment]
released 5.2.1 today.
